### ISSUE TITLE
Replace "fresh apps" weekly email with trending apps by score

### ISSUE THREAD

**Issue by markmhx** [blockstack/app.co#89]
Every Monday, instead of sending the current "Fresh Dapps on App.co" email, we should send a "Hottest trending apps on App.co" email that features that five top apps to rise in popularity via their composite score in the past week.

**Comment by markmhx** [comment]
@cuevasm what do you think about replacing the current email here with this one?

**Comment by cuevasm** [comment]
If it can be done via data from the RSS feed = easy, sure. If not, we need to look at something like SendGrid to pull in data tp that way so we can do more with it. MailChimp is a little basic in this regard so the more complex we want the automation to more we need to think about other tools. I'm fine with that personally, just wanted you to know it's a little bit more effort. How can this data be accessed? Is there an API?

**Comment by hstove** [comment]
I can definitely make an RSS feed that lists in order of ‘hotness’. We might have to do something hacky to make it work with Mailchimp, so they can re-send the same app more than once. Not sure if they match on the ID or if they just include all posts in the last X days.

**Comment by cuevasm** [comment]
Yeah I'm 95% it's based on days, I've never seen the notion of an ID - perhaps spoofing the date in the feed to always be that week. The other problem is that MailChimp only allows you to pull one feed per email so a separate feed would be problematic.

**Comment by cuevasm** [comment]
There is a service to combine feeds that might work - I would need to look into this more to be sure what the possibilities are.

**Comment by hstove** [comment]
Well I can 'combine' the two feeds, but then the issue is how to split them out in the design of the Mailchimp email.

**Comment by cuevasm** [comment]
Yep. Can I get a look at what the API you mentioned offers? Do you plan to add this info in there? If we can just setup our own email to call certain things every week from that, it's probably the more sustainable solution.

**Comment by hstove** [comment]
Our API, as of just now, includes raw web traffic data from similar web, i.e. visits last month. Soon, we'll make a combined metric that uses both web traffic and social metrics to turn that into one single 'hotness' metric. Then, you'd sort by that, and the top X apps would be the 'top X hottest apps right now'.

If you meant the RSS feed, I could make an RSS feed that just sorts by this hotness, but then we wouldn't be able to do multiple RSS feeds due to the Mailchimp limitation.
